### DISCUSSION TITLE
How Better xCloud works with the new native Microsoft keyboard & mouse ?

### DISCUSSION THREAD

**Discussion by yo-hansolo** [redphx/better-xcloud#481] [General]
hello,

I occasionally use the xCloud and I don't understand how Better xCloud works since Microsoft officially supports M&K.
Specifically, as my keyboard is AZERTY, I remapped the Better xCloud keys to be able to play Persona 3 correctly. 
Today the game still works well to simulate QWERTY.

A little setting to handle AZERTY would be cool in my opinion.

I wanted to play Humanity today, which natively supports M&K, and from my AZERTY keyboard, I end up with QWERTY support... I've tried to remap the keyboard configuration but nothing works on the Better xCloud side.

I also don't understand how the keys are mapped in a native Microsoft configuration...

What should I do ? (Except connect my gamepad ^^)
Thanks in advance

**Comment by redphx** [comment]
xCloud's MKB only supports QWERTY layout
